src/components/TextEditor.vue

```diff
@@ -92,6 +92,11 @@ export default {
 			},
 		}
 	},
+	watch: {
+		value(newVal) {
```

Comment:
why do we need this?

Reply:
This is for us to be able to reset the text body.  Otherwise, when calling `reset()` the `bodyVal` in `Composer` is set to '' but the change won't be reflected to the actual text editor